src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +13,9 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+
+        UserService userService = new UserService();
```

Comment:
It is better to move this line to the class level. Becouse otherwise a new object of the type UserService will be created each time this method is called